packages/backend-core/src/cache/invite.ts

```diff
@@ -1,7 +1,10 @@
-import { utils, tenancy, redis } from "../"
+import * as utils from "../utils"
+import { Duration, DurationType } from "../utils"
 import env from "../environment"
+import { getTenantId } from "../context"
+import * as redis from "../redis/init"
 
-const TTL_SECONDS = 60 * 60 * 24 * 7
+const TTL_SECONDS = Duration.fromDays(7).to(DurationType.SECONDS)
```

Comment:
<3

Could we create helpers like `toSeconds` to ease writing this? `Duration.fromDays(7).toSeconds()`?